### PR TITLE
EventCard pagination

### PR DESCRIPTION
**Pull request by Lhyejin**
#60
- Implement Event Card Pagination When scrolling.

### CODE REVIEW

File: web/src/components/event-cards.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ import {
   withStyles,
   ThemeProvider,
 } from '@material-ui/core/styles';
-import PropTypes from 'prop-types';
+// import PropTypes from 'prop-types';
```
Comment (shberry) [inline web/src/components/event-cards.js]: remove commented out lines?

File: web/src/components/event-cards.js [hunk 2]
```diff
@@ -104,17 +124,16 @@ class EventCards extends Component {
...
     const {classes} = this.props;
     const cards = [];
-
     for (let cardIndex = 0; cardIndex < this.props.events.length; cardIndex++) {
-      if (cardIndex % 3 === 0) {
-        cards.push([]);
-      }
+      // if (cardIndex % 3 === 0) {
+      //   cards.push([]);
+      // }
```
Comment (shberry) [inline web/src/components/event-cards.js]: Remove commented out lines?

File: web/src/pages/landing.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import EventCardsQuery from '../components/event-cards-query';
 import Grid from '@material-ui/core/Grid';
 import LandingDescription from '../components/landing-description';
-import PropTypes from 'prop-types';
+// import PropTypes from 'prop-types';
```
Comment (shberry) [inline web/src/pages/landing.js]: Remove commented out lines?

File: web/src/components/event-cards.js [hunk 4]
```diff
@@ -92,6 +92,26 @@ class EventCards extends Component {
...
+  componentDidMount() {
+    window.addEventListener("scroll", this.handleScroll);
+  }
+
+  // Remove Scroll Event Listener
+  componentWillUnmount() {
+    window.removeEventListener("scroll",this.handleScroll);
+  }
+
+  // Handler Scroll Event
```
Comment (lsh9034) [inline web/src/components/event-cards.js]: Could you change this to "Scroll Event Handler"?

File: web/src/components/event-cards.js [hunk 5]
```diff
@@ -104,17 +124,16 @@ class EventCards extends Component {
...
+      // }
       // Push each card component in cards
-      cards[cards.length - 1].push(
-        <Grid item xs={12} sm={6} md={4}>
+      cards.push(
+        <Grid item xs={12} sm={6} md={4} key={this.props.events[cardIndex].id}>
           <Card className={classes.card}
-                key={this.props.events[cardIndex].id}
                 value={this.props.events[cardIndex].id}
+                key={this.props.events[cardIndex].id}
```
Comment (lsh9034) [inline web/src/components/event-cards.js]: Add space back and front of '='

File: web/src/components/event-cards.js [hunk 6]
```diff
@@ -147,7 +166,7 @@ class EventCards extends Component {
     const decks = [];
     for (let cardIndex = 0; cardIndex < cards.length; ++cardIndex) {
       decks.push(
-        <Grid container xs={12} justify='space-around'
+        <Grid container justify='space-around'
               className={classes.cards}>
```
Comment (lsh9034) [inline web/src/components/event-cards.js]: Add space back and front of '='

File: web/src/components/event-cards.js [hunk 7]
```diff
@@ -156,7 +175,10 @@ class EventCards extends Component {
 
     return (
       <ThemeProvider theme={theme}>
-        {decks}
+        <Grid container justify='space-around'
+              className={classes.cards}>
```
Comment (lsh9034) [inline web/src/components/event-cards.js]: this too

File: web/src/components/eventlist-filters.js [hunk 8]
```diff
@@ -92,7 +92,7 @@ class EventlistFilters extends Component {
                         onChange={this.HandlerTagButton}
                         className={classes.buttonShape}
                         value={tagIndex}
-                        label={this.props.filterNames[tagIndex].id}
+                        key={this.props.filterNames[tagIndex].id}
```
Comment (lsh9034) [inline web/src/components/eventlist-filters.js]: this too

File: web/src/components/event-cards.js (hunk 5 above)
Comment (Lhyejin) [reply web/src/components/event-cards.js]: I think that spaces in sides of '=' are not necessary. is that google style?

**Pull request merged**